frontend/themes/ce-demo/components/vaadin-dialog-overlay.css

```diff
@@ -1,7 +1,19 @@
 :host([theme~="no-padding"]) [part="content"] {
   padding: 0;
 }
-/* TODO: TEMPORARY FIX UNTIL https://github.com/vaadin/collaboration-engine-internal/pull/548 IS MERGED AND RELEASED */
-[part="overlay"] {
+
+:host([theme~="editor-view-dialog"]) [part="overlay"] {
+  height: 100%;
+  max-width: 400px;
+  width: 100%;
+  /* Temporary fix https://github.com/vaadin/collaboration-engine-internal/pull/548 */
   z-index: -1;
 }
+
+@media (min-width: 1024px) {
+  :host([theme~="editor-view-dialog"]) [part="overlay"] {
+    height: auto;
+    max-width: none;
+    width: 800px;
+  }
+}
```

Comment:
Based on my experience it would be more conventional to make the media query for the mobile view (using `@media (max-width)`). So the default styles would be for desktop, and the additional mobile styles would be the "exception".

In practice it would also reduce the lines of code, as it would be redundant to override these `height` and `max-width` back to their default values.

Reply:
My experience is that the recommendation is that you should start with something that works in the smallest screen and scale up from there. This because it is easier to make use of additional space, instead of trying to cram a lot of stuff into a small space. 

@anezthes created this, and I guess he is following some best practices on responsive web design.

Reply:
> it is easier to make use of additional space, instead of trying to cram a lot of stuff into a small space

Ok, that's a good point!

Reply:
> Based on my experience it would be more conventional to make the media query for the mobile view

This is the way I've approached things in the past as well. However, with the utility classes being based on Tailwind - which is mobile first - I'm trying to embrace their way of doing things.